index.html

```diff
@@ -41,4 +41,6 @@ <h1 class="section-title">Hey There.<br>I'm Aman</h1>
 
 </body>
 
-</html>
\ No newline at end of file
+
```

Comment:
- [ ] In the toolbar, It would be nice if the hamburger button on the right, is placed in a `nav` tag. This would encourage the use of semantic html tags and also improve code readability. The code structure would turn out as below

```
<header>
   <a href="#"> -- logo goes here -- </a>
   <nav> -- menu icon goes here -- </nav>
</header>
```

- [ ] In the headline, I kindly request that the subtitle uses a heading tag - (I'm a software developer) `hx` (examples: `<h1>,<h2>,<h3>`, etc...) as per project requirements.

- [ ] Please consider styling the list of icons as seen in the Figma design because to fulfill the project requirements.

<img width="293" alt="yyy" src="https://user-images.githubusercontent.com/4247740/144298379-9cf58076-899d-46fd-a6f0-323edd5350fd.PNG">

- [ ] I also highly recommend that the section uses the font and colors (background and font) defined for the template

- [ ] In your design the elements are aligned in the center which is not true if compared with the figma design. I request that you work on that incident. In this project we are trying a smuch as possible to make our designs look more like the figma design 😄.